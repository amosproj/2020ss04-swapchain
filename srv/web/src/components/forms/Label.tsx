import React from "react"

type Props = {
    label: string
}

export const Label = (props: Props) => {
    return (
        <label className="text-xs font-medium leading-4 tracking-wider text-gray-700 uppercase">
            {props.label}</label>
    )
}