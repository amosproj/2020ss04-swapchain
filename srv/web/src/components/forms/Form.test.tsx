import React from 'react';
import {Label} from './Label';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<Label main={<div>main</div>} footer={<h1>footer</h1>}></Label>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});