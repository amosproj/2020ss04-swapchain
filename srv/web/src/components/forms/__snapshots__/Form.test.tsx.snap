// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<label
  className="text-xs font-medium leading-4 tracking-wider text-gray-700 uppercase"
/>
`;
